src/query/interpreter.cpp

```diff
@@ -1521,6 +1548,28 @@ PreparedQuery PrepareFreeMemoryQuery(ParsedQuery parsed_query, const bool in_exp
       RWType::NONE};
 }
 
+PreparedQuery PrepareShowConfigQuery(ParsedQuery parsed_query, const bool in_explicit_transaction) {
+  if (in_explicit_transaction) {
+    throw ShowConfigModificationInMulticommandTxException();
+  }
+
+  auto callback = HandleConfigQuery();
+
+  return PreparedQuery{std::move(callback.header), std::move(parsed_query.required_privileges),
+                       [callback_fn = std::move(callback.fn), pull_plan = std::shared_ptr<PullPlanVector>{nullptr}](
```

Comment:
`std::move(callback.fn)` is redundant. Why are you moving? The lambda does not have a state and therefore is trivially copyable.

Reply:
Actually I think I am wrong on this one because fn is a `std::function` which dynamically allocates memory. I will take a look in the morning, I am from my phone :D

Reply:
Yes, you are wrong. I think std::move here is completely fine.